ixmp/tests/backend/test_jdbc.py

```diff
@@ -63,44 +64,88 @@ def test_close(test_mp_f, capfd):
 VE = pytest.mark.xfail(raises=ValueError)
 
 
-@pytest.mark.parametrize(
-    "args, kwargs, expected",
-    (
-        # Advertised forms
+class TestJDBCBackend:
+    @pytest.fixture(scope="class")
+    def klass(self):
```

Comment:
Maybe `jbc` or `jbclass` is more intuitive? But possible just me :)

Reply:
I was trying to follow the style of the Python standard library and upstream classes which use either [`cls`](https://docs.python.org/3/library/inspect.html#inspect.getmro) or [`klass`](https://docs.python.org/3/library/logging.html#logging.setLoggerClass) for a variable that will be a reference to a class (because `class` is a special keyword that can't be used).

Reply:
Ah, thank you!